src/Business Foundation/App/NoSeries/src/Setup/NoSeriesLine.Table.al

```diff
@@ -0,0 +1,279 @@
+                    exit;
+                if "Allow Gaps in Nos." then
+                    NoSeriesMgt.RecreateSequence(Rec)
+                else begin
+                    "Last No. Used" := NoSeriesMgt.GetLastNoUsed(xRec);
+                    NoSeriesMgt.DeleteSequence(Rec);
+                    "Starting Sequence No." := 0;
+                    "Sequence Name" := '';
+                end;
+                if "Line No." <> 0 then
+                    Modify();
+            end;
+        }
+        field(12; "Sequence Name"; Code[40])
+        {
+            Caption = 'Sequence Name';
+            DataClassification = SystemMetadata;
+            Editable = false;
+        }
+        field(13; "Starting Sequence No."; BigInteger)
+        {
+            Caption = 'Starting Sequence No.';
+            DataClassification = CustomerContent;
+            Editable = false;
+        }
+        field(10000; Series; Code[10]) // NA (MX) Functionality
+        {
+            Caption = 'Series';
+            ObsoleteReason = 'The No. Series module cannot reference tax features.';
+#if not CLEAN24
+            ObsoleteState = Pending;
+            ObsoleteTag = '24.0';
+#else
+            ObsoleteState = Removed;
+            ObsoleteTag = '27.0';
+#endif
+        }
+        field(10001; "Authorization Code"; Integer) // NA (MX) Functionality
+        {
+            Caption = 'Authorization Code';
+            ObsoleteReason = 'The No. Series module cannot reference tax features.';
+#if not CLEAN24
+            ObsoleteState = Pending;
+            ObsoleteTag = '24.0';
+#else
+            ObsoleteState = Removed;
+            ObsoleteTag = '27.0';
+#endif
+        }
+        field(10002; "Authorization Year"; Integer) // NA (MX) Functionality
+        {
+            Caption = 'Authorization Year';
+            ObsoleteReason = 'The No. Series module cannot reference tax features.';
+#if CLEAN24
+            ObsoleteState = Removed;
+            ObsoleteTag = '27.0';
+#else
+            ObsoleteState = Pending;
+            ObsoleteTag = '24.0';
+
+            trigger OnValidate()
+            begin
+                if StrLen(Format("Authorization Year")) <> 4 then
+                    Message(ShouldBeValidYearErr);
+            end;
+#endif
+        }
+        field(12100; "No. Series Type"; Enum "No. Series Type")
+        {
+            Caption = 'No. Series Type';
+            FieldClass = FlowField;
+            CalcFormula = lookup("No. Series"."No. Series Type" where(Code = field("Series Code")));
+            Editable = false;
+        }
+    }
+
+    keys
+    {
+        key(Key1; "Series Code", "Line No.")
+        {
+            Clustered = true;
+        }
+        key(Key2; "Series Code", "Starting Date", "Starting No.", Open)
+        {
+        }
+        key(Key3; "Starting No.")
+        {
+        }
+        key(Key4; "Last Date Used")
+        {
+        }
+    }
+
+    fieldgroups
```

Comment:
Maybe if no fieldgroups then delete the part? But maybe there should be at least one group for consistent.